packages/web3-utils/src/utils.js

```diff
@@ -333,7 +333,11 @@ var toHex = function (value, returnType) {
     if (_.isBoolean(value)) {
         return returnType ? 'bool' : value ? '0x01' : '0x00';
     }
-
+    
```

Comment:
Thanks for proposing this PR! Could you update the JS funDoc on line 322 and extend the CHANGELOG.md file in the root folder of the current branch.